content/docs/command-reference/move.md

```diff
@@ -164,3 +164,48 @@ $ tree
     │   └── foo
     └── data3.dvc
 ```
+
+## Example: Moving ignored files
+
+`dvc move` raises an error if it finds a file with pattern available in
+`.dvcignore` file.
+
+Let's imagine it with following scenerio:
```

Comment:
The idea for this example is pretty good. Let's also review https://dvc.org/doc/user-guide/dvcignore#example-moving-ignored-data please. After reading it now it seems like it may be outdated or it could be improved in its explanation and extended with `dvc move`.

Reply:
I guess this had not much to do with this PR or the issue in question. Out of scope, never mind.